.github/workflows/validate-pipelines.yml

```diff
@@ -40,6 +26,22 @@ jobs:
                   - 'gocd/**'
                   - '.github/workflows/validate-pipelines.yml'
 
+    render:
+        if: needs.files-changed.outputs.gocd == 'true'
```

Comment:
Q: how does this know what files generate output?

Reply:
getsentry/paths-filter `with.filters.gocd` - see right above